src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeRetrieverService.java

```diff
@@ -31,14 +30,14 @@ public EnvelopeRetrieverService(
         this.envelopeAccessProperties = envelopeAccessProperties;
     }
 
-    public List<Envelope> getProcessedEnvelopesByJurisdiction(final String serviceName) {
+    public List<Envelope> findByServiceAndStatus(String serviceName, Status status) {
         log.info("Fetch requested for envelopes for service {}", serviceName);
 
         String jurisdiction = getJurisdictionByServiceName(serviceName);
 
-        log.info("Fetching all processed envelopes for service {} and jurisdiction {}", serviceName, jurisdiction);
```

Comment:
👏

Reply:
Shouldn't the log statement include the status?

Reply:
fixed